dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/statelisteners/PublicStateManagerIT.java

```diff
@@ -120,11 +123,13 @@ private Tag getFakeTag(boolean verified) throws IOException {
 
 
     @Test
-    public void addAnEntry() throws IOException {
+    public void addAnEntry() throws Exception {
```

Comment:
Was like this already, but how did we end up with two PublicStateManagerIT classes with some of the same tests?
Possible merge conflict resolved incorrectly?

Reply:
Hmmm, so it looks like `dockstore-integration-testing/src/test/java/io/dockstore/webservice/statelisteners/PublicStateManagerIT.java` is the one that restarts elasticsearch. 

Maybe move any extra content into that one and delete the other one.  Only seems to save us a few seconds, but its the principle of it.